### PR TITLE
Feature/deploy site

### PR DESCRIPTION
**Pull request by Jayquintana**
# Description
add styling to writing page, and play button element 
Please check the type of change your PR introduces:
- [ ] Bugfix (change which fixes an issue)
- [x] Feature (change which adds functionality)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

 
styling inspirations page
 

### Type of change
style
Please check the relevant option.

- [ ] Non-Breaking Change
- [x ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Big Breaking change (this breaks everything and I am sorry)

# Where should reviewers start?
inspirations component

# How Has This Been Tested?

Please check the option(s) that best describe your tests. 

- [ ] New spec files were added
- [ ] Tests were added to existing spec files
- [ ] Some tests were changed
- [ ] No tests were changed
- [ ] All tests were changed (please explain what in God's name happened)

### Tests Description

- Please add further details of your tests here.

# Checklist:

- [ ] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have removed all unused code or commented why I left it
- [ ] I have deployed my changes to a test-heroku and they work
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass locally with my changes

### CODE REVIEW

File: src/App/App.test.js [hunk 1]
```diff
@@ -1,8 +0,0 @@
-// import { render, screen } from '@testing-library/react';
-// import App from './App';
-
-// test('renders learn react link', () => {
-//   render(<App />);
-//   const linkElement = screen.getByText(/learn react/i);
-//   expect(linkElement).toBeInTheDocument();
-// });
```
Comment (philalewis) [inline src/App/App.test.js:8]: If we don't want to run Jest tests, we should just delete this file.

File: src/Components/Inspirations/Inspirations.scss [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+$button-bg-color: rgba( $button-color, .4 );
+$button-stroke: $button-size/30;
+$arrow-base-size: $button-size/5;
+
+.play-button {
+  align-items: center;
+  border: $button-stroke solid $button-color;
+  border-radius: 50%;
+  box-shadow: 0 0 1px 0px rgba(255, 255, 255, 1);
+  cursor: pointer;
```
Comment (philalewis) [inline src/Components/Inspirations/Inspirations.scss]: We should move this line to the .play-button:hover block. It is not showing the pointer when the user hovers over the play arrow.

**Review by philalewis — CHANGES_REQUESTED** [review]
The cursor doesn't change to a pointer when hovering over the play button on the audio. Can you add that, please?

**Review by Alynn022 — APPROVED** [review]
Approved once you take a look at Phil's comments :)

File: src/App/App.test.js (hunk 1 above)
Comment (Jayquintana) [reply src/App/App.test.js:8]: I tried but it gave me a merge conflict so i had to add it back in

File: src/Components/Inspirations/Inspirations.scss (hunk 2 above)
Comment (Jayquintana) [reply src/Components/Inspirations/Inspirations.scss]: all done

**Review by philalewis — APPROVED** [review]

**Pull request merged**